test/conftest.py

```diff
@@ -48,11 +48,12 @@ def pytest_addoption(parser):
     parser.addoption('--build-base-image', '-B', action='store_true')
     parser.addoption('--aws-id')
     parser.addoption('--instance-type')
+    parser.addoption('--accelerator-type')
     parser.addoption('--docker-base-name', default='pytorch')
     parser.addoption('--region', default='us-west-2')
     parser.addoption('--framework-version', default=PyTorch.LATEST_VERSION)
     parser.addoption('--py-version', choices=['2', '3'], default=str(sys.version_info.major))
-    parser.addoption('--processor', choices=['gpu', 'cpu'], default='cpu')
+    parser.addoption('--processor', choices=['gpu', 'cpu', 'eia'], default='cpu')
```

Comment:
conftest.py is not capable of correctly handling processor = 'eia'.

Please reduce the legal choices for --processor to just ['gpu', 'cpu']

Reply:
Done